src/content/developers/tutorials/optimism-std-bridge-annotated-code/index.md

```diff
@@ -0,0 +1,1399 @@
+---
+title: "Optimism Standard Bridge Contract Walk-Through"
+description: How does the standard bridge for Optimism work? Why does it work this way?
+author: Ori Pomerantz
+sidebar: true
+tags: ["solidity", "optimism", "bridge", "layer 2", "l2"]
+skill: intermediate
+published: 2022-03-30
+lang: en
+---
+
+## Introduction {#introduction}
+
+[Optimism](https://www.optimism.io/) is an [Optimistic Rollup](/developers/docs/scaling/optimistic-rollups/#top).g
+Optimistic rollups can process transactions for a much lower price than the Ethereum mainnet (also known as layer 1 or L1) because transactions are only processed by a few nodes, instead of the every node on the network. 
```

Comment:
```suggestion
Optimistic rollups can process transactions for a much lower price than Ethereum Mainnet (also known as layer 1 or L1) because transactions are only processed by a few nodes, instead of every node on the network. 
```